js/common/lib/tensor.ts

```diff
@@ -234,5 +234,71 @@ export interface TensorConstructor {
   // #endregion
 }
 
+export interface ImageFromTensorOptions {
+  format?: 'RGB'|'RGBA'|'BGR'|'RBG';
```

Comment:
since `'RGB'|'RGBA'|'BGR'|'RBG'` is used multiple times, it is recommended to use a type alias to refer to this.

```ts
export type ImageFormat = 'RGB'|'RGBA'|'BGR'|'RBG';
```

Then you can refer to this type:
```ts
    format?: ImageFormat;
```

Reply:
The same change can be applied to the following also:

```ts
export type ImageTensorLayout = 'NHWC'|'NCHW';

export type ImageNormalizationOptions = {
   ...
};
```

Reply:
Agreed about the image format, updated the code.
About the vector format, we might want to discuss what will be better since there is some overlap between a few properties. (i.e. ImageTensorLayout and tensorHeight & tensorWidth)